### PR TITLE
 Fixed that a user could log zero hours

### PR DESCRIPTION
**Pull request by DarkMatterMatt**
This is why we don't use strict equality 😂 (`hours` is a string)

Also I should probably have tested it before merging it but I was sick of George logging zero hours

### CODE REVIEW

File: volunteers.js [hunk 1]
```diff
@@ -79,13 +79,13 @@ module.exports = async function startVolunteers() {
...
 
-        if (Math.abs(hours)> 24) {
+        if (Math.abs(hours) > 24) {
           message.reply('I can only add or remove a maximum of 24 hours! ' +
             'Please try again?\n');
           break;
         } 
         
-        if (hours === 0) {
+        if (hours == 0) {
```
Comment (ErnWong) [inline volunteers.js]: I would much prefer it to explicitly convert hours to a number and compare with a strict triple-equals sign, rather than relying on double-equals because double-equals increase the chances of the reader to misinterpret. (It makes the design intention of the code ambiguous - e.g. is it a mistake or is it intended?).

**Review by ErnWong — APPROVED** [review]
Sweet it works! I've commented on the use of double-equals - up to you to change it or not.
